.github/workflows/pr-builder.yml

```
@@ -17,16 +17,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        java-version: [ 8, 11, 17 ]
-
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Adopt JDK 
+      - name: Set up Adopt JDK 21
         uses: actions/setup-java@v2
```

actionlint (1.7.11)

[error] the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)